src/analytics/src/main/scala/osmesa/analytics/oneoffs/StreamingAOIMonitor.scala

```diff
@@ -66,27 +59,30 @@ object StreamingAOIMonitor
               "end-sequence",
               short = "e",
               metavar = "sequence",
-              help = "Ending sequence. If absent, this will be an infinite stream."
+              help = "Ending sequence. If absent, this will default to the sequence for Instant.now()."
             )
             .orNone
 
         (intervalOpt, augmentedDiffSourceOpt, startSequenceOpt, endSequenceOpt).mapN {
           (interval, augmentedDiffSource, startSequence, endSequence) =>
             val appName = "StreamingAOIMonitor"
+            val now = Timestamp.from(Instant.now)
 
             implicit val spark: SparkSession = Analytics.sparkSession("Streaming AOI Monitor")
 
             import spark.implicits._
-            spark.withJTS
             import AOIMonitorUtils._
 
-            val areasOfInterest: List[NotificationData] = queryAreasOfInterest(interval)
-            areasOfInterest.foreach { println(_) }
+            val notifications: List[Notification] = queryNotifications(interval)
+            notifications.foreach { println(_) }
+            val notificationsDf = notifications
+              .toDF("notificationId", "userId", "geom", "name", "email")
+            val aoiIndex = spark.sparkContext.broadcast(AOIIndex(notifications))
```

Comment:
Broadcasting AOIIndex was the solution for the PreparedGeom not serializable issue. This object is read-only, shared across the cluster, and should fit in memory, all the big prereqs for being a broadcast variable.